olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/StreamLayerClient.h

```diff
@@ -162,6 +163,39 @@ class DATASERVICE_READ_API StreamLayerClient final {
    */
   client::CancellableFuture<UnsubscribeResponse> Unsubscribe();
 
+  /**
+   * @brief Downloads message data using data handle.
+   *
+   * Users should use this method to download data only for messages that
+   * includes data handle and the data size greater than 1 MB. Message with data
+   * size less then 1 MB will have the data embedded.
+   *
+   * @param message  The `Message` instance that was retrieved using `Poll`
+   * method.
```

Comment:
> hat was retrieved using `Poll` method

Can the message be retrieved from another method (or will it be possible in the future)?

Reply:
No, it can not